### PR TITLE
Added support for displaying RPM MAVLink message in the main QGroundControl values panel

### PR DESCRIPTION
**Pull request by roman-dvorak**
This pull request adds a new feature to QGroundControl, enabling display value of RPM from MAVLink message on the main application window. With this update, users can now show and monitor up to four RPM values, which proves particularly useful for quadcopters.

To achieve this functionality, a new factView group has been created, allowing users to easily track RPM values in a clear panel within the main QGroundControl interface.

The functionality of the PR and features has been tested both with a custom-built QGC and with PX4 autopilot using TFRPM01 sensors, as well as with the PX4 simulator.

### CODE REVIEW

File: src/FactSystem/FactGroup.cc [hunk 1]
```diff
@@ -116,7 +116,8 @@ FactGroup* FactGroup::getFactGroup(const QString& name)
         factGroup = _nameToFactGroupMap[camelCaseName];
         QQmlEngine::setObjectOwnership(factGroup, QQmlEngine::CppOwnership);
     } else {
-        qWarning() << "Unknown FactGroup" << camelCaseName;
+        qWarning() << "Unknown FactGroup CC" << camelCaseName;
```
Comment (DonLakeFlyer) [inline src/FactSystem/FactGroup.cc:119]: Could you change this to be "Unknown FactGroup (camelCaseName)"? That will make it way more clear.

File: custom-example/qgroundcontrol.qrc [hunk 2]
```diff
@@ -328,7 +328,8 @@
 		<file alias="Vehicle/VehicleFact.json">../src/Vehicle/VehicleFact.json</file>
 		<file alias="Vehicle/VibrationFact.json">../src/Vehicle/VibrationFact.json</file>
 		<file alias="Vehicle/WindFact.json">../src/Vehicle/WindFact.json</file>
-                <file alias="Vehicle/HygrometerFact.json">../src/Vehicle/HygrometerFact.json</file>
+        <file alias="Vehicle/HygrometerFact.json">../src/Vehicle/HygrometerFact.json</file>
```
Comment (DonLakeFlyer) [inline custom-example/qgroundcontrol.qrc:331]: Some sort of tabbing problem here.

**Review by DonLakeFlyer — COMMENTED** [review]
Other than a couple minor things this looks good.

**Comment by roman-dvorak** [comment]
Hi @DonLakeFlyer, I have solved all your comments. Please, look at it again. Thank you

File: src/FactSystem/FactGroup.cc (hunk 1 above)
Comment (DonLakeFlyer) [reply src/FactSystem/FactGroup.cc]: Sorry what I meant was to change to:
```qWarning() << "Unknown FactGroup (camelCaseName)" << camelCaseName;```

**Comment by DonLakeFlyer** [comment]
Can you squash this into a single commit and then if the unit tests pass I'll merge.

**Comment by DonLakeFlyer** [comment]
@roman-dvorak Can you squash,  rebase to master and fix conflicts?

**Comment by DonLakeFlyer** [comment]
@roman-dvorak Fix conflicts and squash?

**Comment by julianoes** [comment]
@roman-dvorak bump, if you could fix the conflicts, we'll get it in.

**Pull request closed without merging**